.ci/update-specs.yml

```diff
@@ -0,0 +1,105 @@
+name: update-specs
+
+title: synchronize schema specs
+
+scms:
+  apm-agent-python:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: apm-agent-python
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+sources:
+  sha:
+    kind: file
+    spec:
+      file: 'https://github.com/elastic/apm-data/commit/main.patch'
+      matchpattern: "^From\\s([0-9a-f]{40})\\s"
+  error.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm-data/main/input/elasticapm/docs/spec/v2/error.json
+  metadata.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm-data/main/input/elasticapm/docs/spec/v2/metadata.json
+  metricset.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm-data/main/input/elasticapm/docs/spec/v2/metricset.json
+  span.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm-data/main/input/elasticapm/docs/spec/v2/span.json
+  transaction.json:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm-data/main/input/elasticapm/docs/spec/v2/transaction.json
```

Comment:
@axw 
Is it okay for this to be a "static list" of files, or do we need something more dynamic here that considers file additions/deletions?

Reply:
What are our other options? The set of files won't change often, but they might still change. I think we could go ahead with this, and look at bundling the schema documents into a Compound Schema Document: http://json-schema.org/understanding-json-schema/structuring.html#bundling

Reply:
Opened https://github.com/elastic/apm-data/issues/15 to improve this

Reply:
Another option could be a shell script that does the cloning and copying of the files. Which feels a bit hacky in the updatecli way, but is certainly possible.

Regardless,  a compound schema sounds really good! This will also account for deletions.

I actually encountered a case where `log.json` was not removed (Also in the current implementation). (See https://github.com/elastic/apm-agent-nodejs/pull/3153#pullrequestreview-1289704768) A single file would fix this behaviour.